src/escpos/config.py

```diff
@@ -74,27 +101,23 @@ def load(self, config_path=None):
         except yaml.YAMLError:
             raise exceptions.ConfigSyntaxError("Error parsing YAML")
 
-        if "printer" in config:
-            self._printer_config = config["printer"]
-            printer_name = self._printer_config.pop("type")
-            class_names = {
-                "usb": "Usb",
-                "serial": "Serial",
-                "network": "Network",
-                "file": "File",
-                "dummy": "Dummy",
-                "cupsprinter": "CupsPrinter",
-                "lp": "LP",
-                "win32raw": "Win32Raw",
-            }
-            self._printer_name = class_names.get(printer_name.lower(), printer_name)
+        self._set_config(config)
 
-            if not self._printer_name or not hasattr(printer, self._printer_name):
-                raise exceptions.ConfigSyntaxError(
-                    f'Printer type "{self._printer_name}" is invalid'
-                )
 
-        self._has_loaded = True
+    def load_yaml_string(self, yaml_string):
```

Comment:
Again, we could add the missing return type annotation:
```suggestion
    def load_yaml_string(self, yaml_string) -> None:
```

Reply:
Done in [bcba002](https://github.com/python-escpos/python-escpos/pull/707/commits/bcba002bfe1f47277d54dd2b5bc451b2adaf1d2a)